packs/aws/sensors/sqs_sensor.py

```diff
@@ -32,9 +34,17 @@ def __init__(self, sensor_service, config=None, poll_interval=5):
                                            poll_interval=poll_interval)
 
     def setup(self):
-        self.input_queue = self._get_config_entry(key='keys ', prefix='sqs_sensor')
+        queues = self._get_config_entry(key='input_queues', prefix='sqs_sensor')
```

Comment:
This sensor is in repo only for 2-3 days. I did initial PR too. I'm not sure if anyone else it used yet. So is this really necessary ?